cartographer_grpc/map_builder_server.h

```diff
@@ -28,37 +30,48 @@ namespace cartographer_grpc {
 
 class MapBuilderServer {
  public:
+  using LocalSlamResultCallback = std::function<void(
```

Comment:
I'd choose a different name because it is different from the one declared by TrajectoryBuilderInterface.

Reply:
Renamed to `LocalSlamSubscriptionCallback`